src/protocols/multi_party_ecdsa/gg_2020/test.rs

```diff
@@ -295,70 +330,81 @@ fn keygen_stage4(
 // This test helper is just a demonstration of the same.
 //
 #[cfg(test)]
-fn keygen_orchestrator(
-    params: Parameters,
-) -> Result<
-    (
-        Vec<Keys>,
-        Vec<SharedKeys>,
-        Vec<GE>,
-        GE,
-        VerifiableSS,
-        Vec<EncryptionKey>,
-        Vec<DLogStatement>,
-    ),
-    ErrorType,
-> {
+fn keygen_orchestrator(params: Parameters) -> Result<KeyPairResult, ErrorType> {
     let participants = (0..(params.share_count as usize))
         .map(|k| k + 1)
         .collect::<Vec<usize>>();
-    let mut party_keys_vec = vec![];
-    let mut bc1_vec = vec![];
-    let mut decom_vec = vec![];
-    let mut h1_h2_N_tilde_vec = vec![];
+    // _l in all the below fields means local.
+    // Just naming them differently than the structure fields so that Rust Analyzer does not
+    // complain about "Shorthand Struct initialization" lint.
+    //
+    // Values to be kept private(Each value needs to be encrypted with a key only known to that
+    // party):
+    // 1. party_keys.u_i
+    // 2. party_keys.dk
+    let mut party_keys_vec_l = vec![];
+    // Nothing private in the commitment values.
+    let mut bc1_vec_l = vec![];
+    // Nothing private in the decommitment values.
+    let mut decom_vec_l = vec![];
+    // Nothing private in the below vector.
+    let mut h1_h2_N_tilde_vec_l = vec![];
     for participant in participants.iter() {
         let (party_keys, bc1, decom, h1_h2_N_tilde) = keygen_stage1(*participant);
-        party_keys_vec.push(party_keys);
-        bc1_vec.push(bc1);
-        decom_vec.push(decom);
-        h1_h2_N_tilde_vec.push(h1_h2_N_tilde);
+        party_keys_vec_l.push(party_keys);
+        bc1_vec_l.push(bc1);
+        decom_vec_l.push(decom);
+        h1_h2_N_tilde_vec_l.push(h1_h2_N_tilde);
     }
-    let mut vss_scheme_vec = vec![];
-    let mut secret_shares_vec = vec![];
+    let mut vss_scheme_vec_l = vec![];
+    // Values to be kept private(Each value needs to be encrypted with a key only known to that
+    // party):
```

Comment:
- I suggest to actually encrypt and decrypt 
- encryption is done using the receiver public key